### PR TITLE
Department and Request Page bugs

### PR DESCRIPTION
**Pull request by LaiqaRafiq**

### CODE REVIEW

File: src/components/sales/Request.vue [hunk 1]
```diff
@@ -158,36 +166,42 @@ export default defineComponent({
...
+      showErrorUser: false,
+      showErrorDelivery: false,
+      showErrorDescription: false,
     };
   },
   computed: {
     companyValidation: function () {
       let errorMessage = null;
-      if (this.companies.length <= 0) {
+      console.log("companies", this.companies.length);
```
Comment (ammarhaider001) [inline src/components/sales/Request.vue]: Remove console log

File: src/components/sales/Request.vue [hunk 2]
```diff
@@ -247,6 +271,9 @@ export default defineComponent({
       if (this.requestSent && this.requestSent.id) {
         this.created = `Request is sent. Request id is ${this.requestSent.id}`;
       }
+      }else{
+        console.log("error");
```
Comment (ammarhaider001) [inline src/components/sales/Request.vue]: Remove logs

File: src/components/admin/Department.vue [hunk 3]
```diff
@@ -227,7 +228,8 @@ export default defineComponent({
     margin: 10px;
   }
   .department-container{
-    max-width: 1140px;
+    // max-width: 1140px;
+    max-width: 1163px;
```
Comment (ammarhaider001) [inline src/components/admin/Department.vue:231]: Why hard coded? 
It won't be responsive.

**Review by ammarhaider001 — CHANGES_REQUESTED** [review]

File: src/components/admin/Department.vue (hunk 3 above)
Comment (LaiqaRafiq) [reply src/components/admin/Department.vue:231]: It was already hard coded I just increase the max-width value

Comment (ammarhaider001) [reply src/components/admin/Department.vue:231]: I know. Please make it dynamic/responsive.

**Review by ammarhaider001 — APPROVED** [review]

**Pull request merged**